datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: DOMAINS_INTRO_ID,
+        title: 'Domains Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Domains</strong> page!
+                </p>
+                <p>
+                    <strong>Domains</strong> are collections of related data assets associated with a specific part of
+                    your organization, such as the <strong>Marketing</strong> department.
+                </p>
+                <p>
+                    Learn more about <strong>Domains</strong>{' '}
+                    <a target="_blank" rel="noreferrer noopener" href="https://datahubproject.io/docs/domains">
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: DOMAINS_CREATE_DOMAIN_ID,
+        selector: `#${DOMAINS_CREATE_DOMAIN_ID}`,
+        title: 'Create a new Domain',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Click here to create a new <strong>Domain</strong>.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: USERS_INTRO_ID,
+        title: 'Users Page',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Welcome to the <strong>Users</strong> page!
+                </p>
+                <p>
+                    There are a few different ways to onboard new <strong>Users</strong> onto DataHub.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: USERS_SSO_ID,
+        title: 'Configuring SSO',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    The preferred way to onboard new <strong>Users</strong> is to use <strong>SSO</strong>. Currently,
+                    DataHub supports OIDC SSO.
+                </p>
+                <p>
+                    Learn more about how to configure SSO{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/authentication/guides/sso/configure-oidc-react/#configuring-oidc-in-react"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: USERS_INVITE_LINK_ID,
+        selector: `#${USERS_INVITE_LINK_ID}`,
+        title: 'Sending Invite Links',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can also generate invite links to onboard new <strong>Users</strong> here, with different invite
```

Comment:
Why are we saying "also"?

Remember, someone may see a single step outside of other step context. Remove also.

Easily share an invite link with your colleagues to onboard them onto DataHub. Optionally assign a Role to anyone who joins using the link.

Reply:
Used the new wording